spec/monitoring_spec.ts

```diff
@@ -54,20 +54,20 @@ describe('monitoring', () => {
 
     it('can display the item in Desk Output when it\'s published in a production desk', () => {
         monitoring.openMonitoring();
-        expect(monitoring.getTextItem(3, 2)).toBe('item6');
-        monitoring.actionOnItem('Edit', 3, 2);
+        expect(monitoring.getTextItem(2, 2)).toBe('item7');
```

Comment:
Why are there changes in this test? From the code it doesn't look like it should affect this.